### PR TITLE
Pull Request for Issue #377

### PR DESCRIPTION
**Pull request by ojrb**
Based on @nenonix recommendation, created a PR for each Issue, this PR solved the issues #377 , In the committed **Update-Office-365-Ip-Webservices (Issue #377)** I only add a paragraph for the reworked script requested

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/hybrid-modern-auth-overview.md [hunk 1]
```diff
@@ -85,6 +85,19 @@ Verify and check these items off your list before you continue:
   - Your SIP domain is added as a Federated domain in Office 365
     
   - All SFB Front Ends must have connections outbound to the internet, to Office 365 Authentication URLs (TCP 443) and well known certificate root CRLs (TCP 80) listed in Rows 56 and 125 of the 'Microsoft 365 Common and Office Online' section of [Office 365 URLs and IP address ranges](urls-and-ip-address-ranges.md).
+  
+- **Skype for Business On-premise in a Hybrid environment with Office 365**
```
Comment (JohanFreelancer9) [inline Enterprise/hybrid-modern-auth-overview.md]:
```suggestion
- **Skype for Business On-premise in a hybrid environment with Office 365**
```

File: Enterprise/hybrid-modern-auth-overview.md [hunk 2]
```diff
@@ -85,6 +85,19 @@ Verify and check these items off your list before you continue:
...
+  - A Skype for Business Server 2019 deployment with all servers running Skype for Business Server 2019.
+  
+  - A Skype for Business Server 2015 deployment with all servers running Skype for Business Server 2015.
+  
+  - A deployment with maximum of 2 different server versions as listed below:
```
Comment (JohanFreelancer9) [inline Enterprise/hybrid-modern-auth-overview.md]:
```suggestion
  - A deployment with a maximum of two different server versions as listed below:
```

File: Enterprise/hybrid-modern-auth-overview.md [hunk 3]
```diff
@@ -85,6 +85,19 @@ Verify and check these items off your list before you continue:
...
+  
+     - Skype for Business Server 2015 and Skype for Business Server 2019
```
Comment (JohanFreelancer9) [inline Enterprise/hybrid-modern-auth-overview.md]:
```suggestion
     - Skype for Business Server 2015 and Skype for Business Server 2019.
```

File: Enterprise/hybrid-modern-auth-overview.md [hunk 4]
```diff
@@ -85,6 +85,19 @@ Verify and check these items off your list before you continue:
...
+     
+  - All SFB servers must have the latest CU, please review the article [Skype for Business Server Update](https://docs.microsoft.com/en-us/skypeforbusiness/sfb-server-updates) for the latest ones and installation instructions.
```
Comment (JohanFreelancer9) [inline Enterprise/hybrid-modern-auth-overview.md]:
```suggestion
  - All Skype for Business servers must have the latest cumulative updates. Please review the article [Skype for Business Server Update](https://docs.microsoft.com/en-us/skypeforbusiness/sfb-server-updates) for the latest cumulative updates and installation instructions.
```

File: Enterprise/hybrid-modern-auth-overview.md [hunk 5]
```diff
@@ -85,6 +85,19 @@ Verify and check these items off your list before you continue:
...
+  
+  - There is no Lync Server 2010 or 2013 on the hybrid environment.
```
Comment (JohanFreelancer9) [inline Enterprise/hybrid-modern-auth-overview.md]:
```suggestion
  - There is no Lync Server 2010 or 2013 in the hybrid environment.
```

File: Enterprise/hybrid-modern-auth-overview.md [hunk 6]
```diff
@@ -116,6 +129,16 @@ Verify and check these items off your list before you continue:
   - SSL Offloading is not configured. SSL termination and re-encryption is supported.
     
   - In the event your environment utilizes a proxy server infrastructure to allow servers to connect to the Internet, be sure all Exchange servers have the proxy server defined in the [InternetWebProxy](https://technet.microsoft.com/en-us/library/bb123716%28v=exchg.160%29.aspx) property.
+  
+- **Exchange Server On-premise in a Hybrid environment with Office 365**
+
+  - If you are using Exchange server 2013, At least one server with the Mailbox and Client Access server roles installed. While it's possible to install the Mailbox and Client Access roles on separate servers, we strongly recommend that you install both roles on each server to provide additional reliability and improved performance
```
Comment (JohanFreelancer9) [inline Enterprise/hybrid-modern-auth-overview.md]:
```suggestion
  - If you are using Exchange Server 2013, at least one server must have the Mailbox and Client Access server roles installed. While it is possible to install the Mailbox and Client Access roles on separate servers, we strongly recommend that you install both roles on each server to provide additional reliability and improved performance.
```

File: Enterprise/hybrid-modern-auth-overview.md [hunk 7]
```diff
@@ -116,6 +129,16 @@ Verify and check these items off your list before you continue:
...
+  
+  - If you are using Exchange server 2016 or newer version, at least one server that has the Mailbox server role installed.
```
Comment (JohanFreelancer9) [inline Enterprise/hybrid-modern-auth-overview.md]:
```suggestion
  - If you are using Exchange Server 2016 or a newer version, at least one server must have the Mailbox server role installed.
```

File: Enterprise/hybrid-modern-auth-overview.md [hunk 8]
```diff
@@ -116,6 +129,16 @@ Verify and check these items off your list before you continue:
...
+  
+  - There is no Exchange server 2007 or 2010 in the Hybrid environment.
```
Comment (JohanFreelancer9) [inline Enterprise/hybrid-modern-auth-overview.md]:
```suggestion
  - There is no Exchange Server 2007 or 2010 in the hybrid environment.
```

File: Enterprise/hybrid-modern-auth-overview.md [hunk 9]
```diff
@@ -116,6 +129,16 @@ Verify and check these items off your list before you continue:
...
+  
+  - All Exchange servers must have the latest CU, review the article [Upgrade Exchange to the latest Cumulative Update](https://docs.microsoft.com/en-us/exchange/plan-and-deploy/install-cumulative-updates?view=exchserver-2019) for the latest ones and installation instructions.
```
Comment (JohanFreelancer9) [inline Enterprise/hybrid-modern-auth-overview.md]:
```suggestion
  - All Exchange servers must have the latest cumulative updates. Review the article [Upgrade Exchange to the latest Cumulative Update](https://docs.microsoft.com/en-us/exchange/plan-and-deploy/install-cumulative-updates?view=exchserver-2019) for the latest cumulative updates and installation instructions.
```

**Comment by ojrb** [comment]
@mypil  I must deleted the file **hybrid-modern-auth-overview.md** because is not part of the PR, please take in account for the review and merge, only the changes to the file **Update Office-365-Ip-Webservices (issue #377)**, Review and merge based on this !

File: Enterprise/hybrid-modern-auth-overview.md (hunk 1 above)
Comment (nenonix) [reply Enterprise/hybrid-modern-auth-overview.md]: Skype for Business **on-premises** in a hybrid environment with Office 365

**Comment by mypil** [comment]
@kelleyvice-msft - Can you please assist to check, approve and merge the content for this PR based on issue #377?

Thank you.

**Comment by mypil** [comment]
@kelleyvice-msft - We would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #377?

Thank you.

**Comment by kelleyvice-msft** [comment]
This has conflicting edits to the same files as issue #392. Both PRs have conflicts. This PR has updates to hybrid-modern-auth-overview.md, while #392 deletes this file. I'm trying to sort out the best way to handle this before I merge either PR.

**Comment by mypil** [comment]
@kelleyvice-msft - Should we just let @ojrb close the 2 conflicting PRs and create just one PR which would cover the feedback from the 2 issues that talks about the same article?

**Comment by mypil** [comment]
@ojrb - Please proceed to close this PR and #392 then just create one PR which would address the feeback of the 2 issues for the same file/article. Thank you.

**Pull request closed without merging**